web/react/components/post_list.jsx

```diff
@@ -459,6 +459,19 @@ module.exports = React.createClass({
             previousPostDay = utils.getDateForUnixTicks(post.create_at);
         }
 
+        if (posts == undefined) {
```

Comment:
This is pretty minuscule and I'm 0/5 on whether you change it but I think it would be a bit better if you changed line 462 here to an 'else' and move the start of the if statement to line 410 (just below the definition of 'postCtls'). Changing the if condition to be negative of what it is now. 

This way we have two clearly defined code paths depending on whether 'posts' is set or not. It also avoids the unlikely possibility that order is set and posts is not causing an error in the loop, and also in the initialization of 'previousPostDay' you can remove the 'posts &&'.